fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraAclIT.java

```diff
@@ -184,4 +185,31 @@ public void testGetNonExistentAcl() throws Exception {
 
     }
 
+    @Test
+    public void testCreateAclWithBody() throws Exception {
+        createObjectAndClose(id);
+
+        final HttpPut put = new HttpPut(subjectUri + "/" + FCR_ACL);
+        final String aclBody = "@prefix acl: <http://www.w3.org/ns/auth/acl#> .\n" +
+                               "@prefix foaf: <http://xmlns.com/foaf/0.1/> .\n" +
+                               "@prefix ldp: <http://www.w3.org/ns/ldp#> .\n" +
+                               "\n" +
+                               "<#authorization_1> a acl:Authorization ;\n" +
+                               "    acl:agentClass foaf:Agent ;\n" +
+                               "    acl:accessToClass ldp:Resource ;\n" +
+                               "    acl:default <https://example.org/root/> ;\n" +
+                               "    acl:mode acl:Read .";
+
+        put.setEntity(new StringEntity(aclBody));
+        put.setHeader("Content-Type", "text/turtle");
+
+        final String aclLocation;
+        try (final CloseableHttpResponse response = execute(put)) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+            aclLocation = response.getFirstHeader("Location").getValue();
+            // verify the acl container is translated to fcr:acl
+            assertEquals(subjectUri + "/" + FCR_ACL, aclLocation);
```

Comment:
We should probably test the content of the returned body to make sure that the triples were indeed added.

Reply:
Ah good catch.